config/environments/production.rb

```diff
@@ -76,4 +76,14 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+
+  config.paperclip_defaults = {
+      storage: :s3,
+      s3_credentials: {
+        bucket: "cardapp",
+        access_key_id: ENV["s3_key"],
+        secret_access_key: ENV["s3_secret"],
+        url: ":s3_domain_url"
+      }
+    }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.